osu.Game/Online/Multiplayer/MultiplayerCountdown.cs

```diff
@@ -15,13 +15,24 @@ namespace osu.Game.Online.Multiplayer
     [Union(1, typeof(ForceGameplayStartCountdown))]
     public abstract class MultiplayerCountdown
     {
+        /// <summary>
+        /// A unique identifier for this countdown.
+        /// </summary>
+        [Key(0)]
+        public int ID { get; set; }
+
         /// <summary>
         /// The amount of time remaining in the countdown.
         /// </summary>
         /// <remarks>
-        /// This is only sent once from the server upon initial retrieval of the <see cref="MultiplayerRoom"/> or via a <see cref="CountdownChangedEvent"/>.
+        /// This is only sent once from the server upon initial retrieval of the <see cref="MultiplayerRoom"/> or via a <see cref="CountdownStartedEvent"/>.
         /// </remarks>
-        [Key(0)]
+        [Key(1)]
         public TimeSpan TimeRemaining { get; set; }
+
+        /// <summary>
+        /// Whether only a single instance of this <see cref="MultiplayerCountdown"/> type may be active at any one time.
+        /// </summary>
+        public virtual bool IsExclusive => false;
```

Comment:
This has a default value of `false`, but all existing usages override it to `true` correct?

Curious as to an example of when this would be `false`, and any rationale behind making that the default.

Reply:
I was imagining generic user countdowns being a possibility, and multiples of them being able to be started at once. I could make it true by default if you prefer.

Reply:
Let's go with default `true` for now then.